_data/library/scalalibs.yml

```diff
@@ -273,3 +273,8 @@
       dep: |
         addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full)
         libraryDependencies += "com.thoughtworks.enableIf" %% "enableif" % "latest.release"
+    - name: sbt-node
+      url: https://github.com/jokade/sbt-node
+      desc: A Sbt plugin for integration with npm / node.js based build tools.
+      dep: 'addSbtPlugin("de.surfice" % "sbt-node" % "latest.release")'
+
```

Comment:
On a side note, I would like to point out that sbt-node's choice of sbt setting names clashes with the already established [scalajs-bundler](https://scalacenter.github.io/scalajs-bundler/getting-started.html). It's not a problem for inclusion in this per se, but you should consider different names.